metaci/release/migrations/0016_auto_20210511_2018.py

```diff
@@ -0,0 +1,28 @@
+# Generated by Django 3.1.7 on 2021-05-11 20:18
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+    def populate_version_number(apps, schema_editor):
```

Comment:
It would be less confusing to put this as a standalone function outside the Migration class, since it doesn't take a `self` param and isn't called as an instance method or class method.